testing/endpoint.go

```diff
@@ -521,6 +532,39 @@ func (endpoint *Endpoint) SendPacket(
 	return sequence, nil
 }
 
+// RecvPacketV2 receives a v2 packet on the assoicated endpoint
+// The counterparty client is updated
+func (endpoint *Endpoint) RecvPacketV2(packet channeltypesv2.Packet) error {
```

Comment:
Should eventually fill these out with ack and timeout as well

Reply:
ditto https://github.com/cosmos/ibc-go/blob/1682792b9428df0df33709b8192c8bae3b508f71/testing/endpoint_v2.go#L32